### PR TITLE
Add flat.social

### PR DESCRIPTION
**Pull request by nazywamsiepawel**
Hi,

Awesome work on collecting all those apps in one place!

Sending a PR with a suggestion to add flat.social to your list too :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ These apps can be run directly in the browser.
 - [Chatmosphere](https://chatmosphere.cc/): Open-source proximity chat prototype
 - [CozyRoom](https://cozyroom.xyz): Peer-to-peer spatial audio environment
 - [CubeChat](https://cubechat.io): Proximity call in 3D
+- [flat.social](https://flat.social): a platform for hosting fun online events and meetings.
```
Comment (billmei) [inline README.md]:
```suggestion
- [flat.social](https://flat.social): A platform for hosting fun online events and meetings
```

**Review by billmei — CHANGES_REQUESTED** [review]
Thanks! Fix typo and you are good to go 👍

**Comment by nazywamsiepawel** [comment]
Thank you! Didn't know you can suggest a change on Github so easily, nice one!

**Review by billmei — APPROVED** [review]
Thanks!

**Pull request merged**